state/migrations/20230913120537_events_missing_previous.sql

```diff
@@ -0,0 +1,10 @@
+-- +goose Up
+ALTER TABLE IF EXISTS syncv3_events
+    ADD COLUMN IF NOT EXISTS missing_previous BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Not sure if this is the right thing to do. Considerations:

- I'd prefer to avoid a nullable column.
- Adding a column with a default should be fast [these days](https://www.postgresql.org/docs/current/ddl-alter.html#DDL-ALTER-ADDING-A-COLUMN).
- Don't want the result of the migration from being different to the "proper" schema in the events table source code.
- I could make the migration drop the default... but that would presumably rewrite the entire table.

Also:
- Not sure the default value is correct here. Arguably TRUE would be more correct (we don't know if these events came from limited syncs; maybe we ought to assume the worst)

Reply:
Default FALSE makes sense to me as historically we always assume we have the previous.